docs/metrics.md

```diff
@@ -60,6 +60,41 @@ metricsConfig: |
 
 ### Prometheus scraping
 
+A metrics service is not installed as part of [the default installation](quick-start.md) so you will need to add one if you wish to use a Prometheus Service Monitor. If you have more than one controller pod, using one as a [hot-standby](high-availability.md), you should use [a headless service](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services) to ensure that each pod is being scraped so that no metrics are missed.
```

Comment:
Please convert to [one sentence per line](https://github.com/argoproj/argo-workflows/blob/bf64aeb92ae7d250a677545dae377585513089c9/docs/doc-changes.md?plain=1#L12) for differencing and patching purposes.